_snippets/integrations/builtin/credentials/google/unverified-app.md

```diff
@@ -1,3 +1,3 @@
-If using the OAuth authentication method, you might see the warning **Google hasn't verified this app**. To avoid this, you can create OAuth credentials from the same account you want to authenticate. 
+If using the OAuth authentication method, you might see the warning **Google hasn't verified this app**. To avoid this, you can add your email to the list of testers for the app. To do so, go to the ["Audience" page](https://console.cloud.google.com/auth/audience) page and add the email you're signing in with to the list of "Test users".
```

Comment:
```suggestion
If using the OAuth authentication method, you might see the warning **Google hasn't verified this app**. To avoid this:

- If your app **User Type** is **Internal**, create OAuth credentials from the same account you want to authenticate.  
- If your app **User Type** is **External**, you can add your email to the list of testers for the app: go to the [**Audience** ](https://console.cloud.google.com/auth/audience) page and add the email you're signing in with to the list of **Test users**.
```